Makefile

```diff
@@ -72,6 +73,12 @@ vet: ## Run go vet against code.
 test: manifests fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
```

Comment:
should we change this to use gotestsum?

https://github.com/kubernetes-sigs/jobset/blob/main/Makefile#L123-L125

Reply:
Make sense, done.